File: appverifier/src/main/java/com/android/mdl/appreader/issuerauth/vical/DataItemDecoderException.java
package com.android.mdl.appreader.issuerauth.vical;

public class DataItemDecoderException extends Exception {

    private static final long serialVersionUID = 1L;

    /**
     * Indicates that a problem occurred during the decoding of a VICAL or CertificateInfo field.
     * @param message the problem indication
     */
    public DataItemDecoderException(String message) {
        super(message);
    }

    /**
     * Indicates that a problem occurred during the decoding of a VICAL or CertificateInfo field.
     * @param message the problem indication
     * @param cause the cause of the problem, never null
     */
    public DataItemDecoderException(String message, Throwable cause) {
        super(message, cause);
    }
}

File: appverifier/src/main/java/com/android/mdl/appreader/issuerauth/vical/DataItemDecoderException.kt
package com.android.mdl.appreader.issuerauth.vical

import java.lang.Exception

class DataItemDecoderException : Exception {
    /**
     * Indicates that a problem occurred during the decoding of a VICAL or CertificateInfo field.
     * @param message the problem indication
     */
    constructor(message: String?) : super(message)

    /**
     * Indicates that a problem occurred during the decoding of a VICAL or CertificateInfo field.
     * @param message the problem indication
     * @param cause the cause of the problem, never null
     */
    constructor(message: String?, cause: Throwable?) : super(message, cause)

    companion object {
        private const val serialVersionUID = 1L
    }
}